runtime/kusama/src/lib.rs

```diff
@@ -1326,6 +1327,97 @@ impl pallet_nomination_pools::Config for Runtime {
 	type MaxPointsToBalance = MaxPointsToBalance;
 }
 
+pub struct RemoveGovV1Storage;
+impl RemoveGovV1Storage {
+	fn get_pallets() -> Vec<&'static str> {
+		vec![
+			"Democracy",
+			"Council",
+			"TechnicalCommittee",
+			"PhragmenElection",
+			"TechnicalMembership",
+			"Treasury",
+		]
+	}
+
+	fn count_keys(prefix: &[u8]) -> u32 {
+		let mut current = prefix.clone().to_vec();
+		let mut counter = 0;
+		while let Some(next) = sp_io::storage::next_key(&current[..]) {
+			if !next.starts_with(prefix) {
+				return counter
+			}
+			counter += 1;
+			current = next;
+		}
+		0
```

Comment:
```suggestion
		unreachable!()
```

right? I am not entirely sure.